### DISCUSSION TITLE
Running a specific test?

### DISCUSSION THREAD

**Discussion by olalonde** [nodejs/help#5044] [Q&A] [answered]
`make test-parallel` is taking a long time. Is it possible to just run a single test while I'm developing? I'm only modifying JS files in `./lib`.

**Comment by cjihrig** [comment, marked as answer]
If you want to run a single test file, you can run `./tools/test.py test/parallel/test-whatever-test.js`. You can also use wildcards like `./tools/test.py test/parallel/test-fs-*`.

**Comment by aduh95** [comment]
Node that `make test-parallel` runs tests one by one by default, what you probably want is `make jstest -j12` (replace 12 by whatever value works for your system, I have a 16-core CPU so that's what I use).
